src/compass/utils/lut.py

```diff
@@ -40,8 +40,8 @@ def cumulative_correction_luts(burst, dem_path,
                                           rg_step=rg_step,
                                           az_step=az_step)
 
-    rg_lut_data = geometrical_steer_doppler.data
-    az_lut_data = bistatic_delay.data + az_fm_mismatch.data
+    rg_lut_data = geometrical_steer_doppler.data * isce3.core.speed_of_light / 2.0
+    az_lut_data = -(bistatic_delay.data + az_fm_mismatch.data)
```

Comment:
Maybe we could add some comments on why we're doing the negation for `az_lut_data` and the conversion to meters for `rg_lut_data`?